components/search-filters/filter-geo.tsx

```diff
@@ -0,0 +1,90 @@
+import axios from 'axios';
+import { useRef, useState } from 'react';
+import { Loader } from '#components-ui/loader';
+import { debounce } from '#utils/helpers/debounce';
+
+interface IGeoSuggest {
+  label: string;
+  value: string;
+}
+
+export const FilterGeo: React.FC<{ cp_dep?: string; cp_dep_label?: string }> =
+  ({ cp_dep, cp_dep_label }) => {
+    const [labelDep, setLabelDep] = useState(cp_dep_label);
+    const [dep, setDep] = useState(cp_dep);
+    const [isLoading, setLoading] = useState(false);
+    const [geoSuggests, setGeoSuggests] = useState([]);
+
+    const geoSearchRef = useRef(null);
```

Comment:
To avoid ts-ignore line 37 it should be typed 

```suggestion
    const geoSearchRef = useRef<HTMLInputElement | null>(null);
```